autokeras/auto/auto_model.py

```diff
@@ -28,30 +30,44 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.inputs = []
         self.outputs = []
+        self._hypermodels = []
+        self._nodes = []
+        self._built = False
         self.tuner = None
+        self.tuner = tuner.SequentialRandomSearch(self, objective=self._get_metrics())
```

Comment:
You should now be able to add Keras Tuner (installed from GitHub) as a dependency of AutoKeras.

In this case, since this is a base class, you shouldn't need to set a concrete tuner.